app/theme/app.ios.scss

```diff
@@ -1,11 +1,10 @@
-
 // http://ionicframework.com/docs/v2/theming/
 
 
 // App Shared Variables
 // --------------------------------------------------
 // Shared Sass variables go in the app.variables.scss file
-@import 'app.variables';
+@import "app.variables";
```

Comment:
Copying what I assume the ionic team considers best practice... see https://github.com/driftyco/ionic2-starter-tabs/commit/27d4ff7893c985372832cc28832ad7492de6a793